activerecord/lib/active_record/core.rb

```diff
@@ -25,6 +25,12 @@ module Core
       # Specifies the job used to destroy associations in the background
       class_attribute :destroy_association_async_job, instance_writer: false, instance_predicate: false, default: false
 
+      ##
+      # :singleton-method:
+      #
+      # Specifies the number of associated records destroyed in a single background job.
```

Comment:
This could be a little more clear that the records will be split between background jobs when set.

Reply:
For sure. I made this a little more descriptive.